### ISSUE TITLE
Links on logos in the navigation and footer point to different locations

### ISSUE THREAD

**Issue by juhakujala** [City-of-Helsinki/kerrokantasi-ui#703]
The logo link in the navigation points on the kerrokantasi homepage but the logo link in the footer points to hel.fi. It's a bit confusing having the same logo on the page that points to two different locations.

Suggestion: Make both logos point to kerrokantasi homepage and add a separate link to hel.fi in the copyright text `2019 Helsingin kaupunki`

**Comment by Rikuoja** [comment]
Hm, I believe they serve two different purposes, so will have to ask @katjahenttonen opinion on this one.

I think the city main web page should be easily accessible from kerrokantasi, not hidden under the copyright clause. Then, of course, we should think what we want to do with the Helsinki logo on the top; currently, it duplicates the function of the Kerrokantasi text below.

**Comment by juhakujala** [comment]
I think nowadays it's more of a rule that the logo in the main navigation takes you to the homepage of the app you are using. Changing this might not be the best idea.

I see your point but maybe we could come up with another solution to access city main website easily?

**Comment by katjahenttonen** [comment]
The header is somewhat standard for all digital services in Helsinki and the logo always goes the front page (see e.g. https://www.myhelsinki.fi, helsinkioppii.fi, varaamo.fi etc) so I agree with @juhakujala  that we should not change that. But always agree with @Rikuoja  that we should have clear link to the city website. I suggest adding a new section to the footer entitled "Linkit or Linkkejä" (a little bit like at uuttahelsinki.fi footer) which could list some links, including one to the city website and others e.g. more e-participation opportunities such as participatory budgeting and general feedback form. The later link has been on my todo-list anyways since now some people use technical feedback email tell their opinions on public toilets and bus routes.

**Comment by katjahenttonen** [comment]
Sovimme palaverissa 9.8. että kaupungin logo ei jatkossa johda mihinkään mutta sen alle tulee linkki kaupungin sivulle tekstinä,  www.hel.fi tai www.turku.fi.  Muiden linkkien lisääminen jätettiin möyhemmäksi.
